bitnami/jupyterhub/templates/hub/configmap.yaml

```
@@ -0,0 +1,505 @@
+{{- if not .Values.hub.existingConfigmap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: hub
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}-hub
```

We should start validation we're not exceeding names limits when appending suffixes:
```suggestion
  name: {{ printf "%s-hub" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
```